ckan/logic/action/create.py

```diff
@@ -296,6 +296,12 @@ def resource_create(context, data_dict):
 
     upload = uploader.get_resource_uploader(data_dict)
 
+    if not 'size' in data_dict and hasattr(upload, 'upload_file'):
```

Comment:
@TkTech , I tried to follow the same code style as in the CKAN code.
From the same method above my code:
`if not 'resources' in pkg_dict:`